english/pinch/go-out/keep-running/keep-running.md

```diff
@@ -1,3 +1,10 @@
 You keep running, going nowhere in particular.
 
-Then you tripped over something and fell flat on your face.
\ No newline at end of file
+Then you tripped over something and fell flat on your face.
+
+You tripped on a half-open manhole cover. You move the cover to 
+reveal the top of a ladder leading down into the dark.
+
+[Climb down](../../../zork/zork.md)
+
+[Notice that there's a patch of fluffy moss off to your left and go to take a nap?](english/explore-outside/moss-nap/moss-nap.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.